pkg/resourceprovider/provider.go

```diff
@@ -331,7 +339,14 @@ func (p *resourceProvider) GetNodeMetrics(nodes ...*corev1.Node) ([]metrics.Node
 	// run the actual query
 	qRes := p.queryBoth(now, nodeResource, "", nodeNames...)
 	if qRes.err != nil {
-		klog.Errorf("failed querying node metrics: %v", qRes.err)
+		if promErr, ok := qRes.err.(*client.Error); ok {
+			klog.Errorf("failed querying node metrics: %s", promErr.Error())
```

Comment:
ditto: Log all the info we have here, i.e. status code, query, error message, type